### PR TITLE
fixes to mappings script

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?

This makes the es-mappings script compile.

It doesn't work yet.

## Tested?
- [ ] locally
- [ ] on TEST

### CODE REVIEW

File: dev/script/es-ssh-ssm-tunnel.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ELB_NAME=""
+
+echo "🛰 fetching connection details from ssm"
+
+SSM_COMMAND=$(ssm ssh --profile media-service -t elasticsearch-data,grid-elasticsearch,$STAGE --newest --ssm-tunnel --raw)
+
+echo "📠 ESTABLISHING CONNECTION"
+
+echo "$SSM_COMMAND"
+eval $SSM_COMMAND $VERBOSE -L 9300:localhost:9200 -N
```
Comment (akash1810) [inline dev/script/es-ssh-ssm-tunnel.sh:53]:
```suggestion
eval $SSM_COMMAND $VERBOSE -L 9300:localhost:9300 -N
```

File: dev/script/es-ssh-ssm-tunnel.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+function HELP {
+>&2 cat << EOF
+
+  Usage: ${0} -t ES-HOST [-s TEST] [-u ubuntu]
+
+  This script sets up an ssh tunnel from localhost port 9300 to the ElasticSearch provided on port 9300
```
Comment (akash1810) [inline dev/script/es-ssh-ssm-tunnel.sh:9]: Why 9300 over 9200?

File: dev/script/es-ssh-ssm-tunnel.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+done
+shift $((OPTIND-1))
+
+
+if [ -z "${STAGE}" ]; then
+  STAGE="TEST"
+fi
+echo "📡 getting ELB name from AWS"
+
+ELB_NAME=""
```
Comment (akash1810) [inline dev/script/es-ssh-ssm-tunnel.sh:44]: should this be the empty string?

File: dev/script/es-ssh-ssm-tunnel.sh (hunk 2 above)
Comment (AWare) [reply dev/script/es-ssh-ssm-tunnel.sh:9]: I think it was in the README or something equally not good as an answer. Will fix.

File: dev/script/es-ssh-ssm-tunnel.sh (hunk 3 above)
Comment (AWare) [reply dev/script/es-ssh-ssm-tunnel.sh:44]: This step isn't needed as I got the direct tunnel working, will remove.

**Comment by akash1810** [comment]
Could we update the CI step to ensure we catch this class of regression earlier in the future?

**Comment by jonathonherbert** [comment]
Great point @akash1810, me and @AWare discussed this on Friday and I'd forgotten it over the weekend. This would be 💯

**Pull request merged**